README.md

```diff
@@ -10,7 +10,7 @@
 * Simple API
 * "Store" approach for backend
 * Redis support (but not tied too)
-* Middlewares: HTTP and [Gin][4]
+* Middlewares: HTTP, [Gin][4] and [Beego][6]
```

Comment:
We do not support `Beego` middleware per-say, since it doesn't need a middleware driver (https://github.com/ulule/limiter/tree/master/drivers/middleware).

However, I'm fine with a link of your example :slightly_smiling_face:

Reply:
Fixed